Example/Example/CharactersViewController.swift

```diff
@@ -24,4 +24,13 @@ class CharactersViewController: ExampleViewController, BothamTableViewController
         pullToRefreshHandler.addTo(tableView)
         super.viewDidLoad()
     }
+    
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        pullToRefreshHandler.beginRefreshing()
+        let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(3 * Double(NSEC_PER_SEC)))
+        dispatch_after(delayTime, dispatch_get_main_queue()) {
+            self.pullToRefreshHandler.beginRefreshing()
+        }
```

Comment:
I was testing manually to see if it worked programatically and forgot remove that code, in the next commit this code is removed. Yeah, i can do a test, no problem.  @pedrovgs Basically the behaviour i was testing is programatically start refreshing and after a short period stoping it, so i did with that code instead of having to create buttons. (Image a list that every time you enter is refreshed automatically, like a feed).